modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/string.ts

```diff
@@ -12,6 +12,24 @@
  * details.
  */
 
+const defaultLanguageId = Liferay.ThemeDisplay.getDefaultLanguageId();
+
+type Label = {
+	[key: string]: string;
+};
```

Comment:
Hey @alinedoleron, We don't need to create a new type for this, we can use an existing type in the Label which is: `LocalizedValue<string>`

Reply:
Thanks for the suggestions, Dan!